site/index.rst

```diff
@@ -83,7 +83,7 @@ YouTube
     student-lectures/Differential_Evolution/Differential_Evolution
     student-lectures/Search-Space-Analysis/Search_Space_Analysis
     student-lectures/ant-colony/ant-colony-optimization
-    student-lectures/coevolution/coevolution
```

Comment:
what is this change?

Reply:
That was to fix an error noticed with another student's lecture that meant it wasn't showing in the sidebar and was showing as an error/warning when building on my computer and I noticed it wasn't showing on the site either